spec/index.html

```diff
@@ -789,18 +789,19 @@ <h3>Literals</h3>
       if and only if the following are all true:</p>
 
     <ul>
-      <li>The two <a>lexical forms</a> compare equal.</li>
-      <li>The two <a>datatype IRIs</a> compare equal.</li>
-      <li>The two <a>language tags</a> are either both absent, or both present and compare equal.</li>
+      <li>The two <a>lexical forms</a> compare equal,
+        where this comparison is performed using
+        <a data-cite="I18N-GLOSSARY#dfn-case-sensitive">case-sensitive matching</a>
+        (see description of string comparison in
+        <a href="#rdf-strings" class="sectionRef"></a>).</li>
+      <li>The two <a>datatype IRIs</a> compare [=IRI equality|equal=]
+        (per [=IRI equality=]).</li>
+      <li>The two <a>language tags</a> are either both absent, or both present and compare equal,
+        where this comparison is performed using
+        <a data-cite="I18N-GLOSSARY#dfn-case-sensitive">ASCII case-insensitive matching</a>
```

Comment:
This is a link to "case-sensitive", same as the link in "lexical forms".

```suggestion
        <a data-cite="I18N-GLOSSARY##dfn-ascii-case-insensitive">ASCII case-insensitive matching</a>
```

Reply:
Other than that +1.

Reply:
@afs  Good catch! I simply moved this link from the paragraph that was below, and didn't even realize that the link was wrong there already.